packages/suite/src/hooks/suite/useOpenSuiteDesktop.ts

```diff
@@ -0,0 +1,26 @@
+import TrezorConnect from '@trezor/connect';
+import { useWindowFocus } from '@trezor/react-utils';
+import { SUITE_BRIDGE_DEEPLINK, SUITE_URL } from '@trezor/urls';
+import { isWebUsb } from 'src/utils/suite/transport';
+import { useSelector } from 'src/hooks/suite';
+
+export const useOpenSuiteDesktop = () => {
+    const transport = useSelector(state => state.suite.transport);
+    const windowFocused = useWindowFocus();
+    const handleOpenSuite = () => {
+        if (isWebUsb(transport)) {
+            TrezorConnect.disableWebUSB();
```

Comment:
hmm I am brainstorming (not related to this code here). The problem: https://github.com/trezor/trezor-suite/issues/13179

We at least used to have manual solution - disable webusb button. It was not perfect and I think we should address it now. The right solution would be to disable webusb somewhere under the hood based on either bridge polling, or when a connect call ends with (specific) error.